flink-runtime/src/test/java/org/apache/flink/runtime/operators/testutils/DriverTestBase.java

```diff
@@ -190,18 +190,18 @@ protected void setNumFileHandlesForSort(int numFileHandles) {
         this.numFileHandles = numFileHandles;
     }
 
-    @SuppressWarnings("rawtypes")
+    @SuppressWarnings({"rawtypes", "unchecked"})
     protected void testDriver(Driver driver, Class stubClass) throws Exception {
```

Comment:
```suggestion
     /**
     * @deprecated Use {@link #testDriver(Driver, Function)} instead.
     */
    @Deprecated
```
I'm wondering whether we should make this method deprecated and suggest using your newly added method. Passing the class doesn't bring any benefits because we call the default constructor internally anyway. :shrug: 

The only downside of that one would be that all the old test classes are using this method and would have a deprecation warning being add :roll_eyes: